### PR TITLE
Closes #1001 #971: Fixing mails translations

### LINKED ISSUE #1001: In Polish interface, password reset e-mails are sent in Ponglish
Interface language: PL
Prefferred language for e-mails: EN

Result: 

Cześć Wojtek Nowak,

Ktoś - mamy nadzieję, że Ty - poprosił o reset hasła do wojteknowak.

.Kliknij na link znajdujący się poniżej, by zresetować hasło do Twojego konta w repozytorium:

.../passwordreset.xhtml?token=00160e3b-8941-4fbc-ab28-ef7ac513b4a1

Powyższy link będzie aktywny tylko przez następne 60 minut.

Skontaktuj się z nami, jeśli nie prosiłeś o reset hasła lub potrzebujesz dodatkowej pomocy.

You may contact us for support at ....

Thank you,
RepOD Pomoc

### PR DESCRIPTION
**Pull request by dbojanek**

### CODE REVIEW

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/confirmemail/ConfirmEmailUtil.java [hunk 1]
```diff
@@ -53,4 +54,35 @@ public static String friendlyExpirationTime(long expirationLong) {
         }
     }
 
+    public static String friendlyExpirationTime(long expirationLong, Locale language) {
```
Comment (MrK191) [inline dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/confirmemail/ConfirmEmailUtil.java:57]: Is method without Locale working correctly (we don't want locale in every instance)?.

File: dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/BrandingUtil.java [hunk 2]
```diff
@@ -23,6 +24,20 @@ public static String getSupportTeamName(InternetAddress systemAddress, String ro
         return BundleUtil.getStringFromBundle("contact.support", Arrays.asList(saneDefault));
     }
 
+    public static String getSupportTeamName(InternetAddress systemAddress, String rootDataverseName, Locale locale) {
```
Comment (MrK191) [inline dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/BrandingUtil.java:27]: Is method without Locale working correctly (we don't want locale in every instance)?.

**Review by MrK191 — CHANGES_REQUESTED** [review]

File: dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/confirmemail/ConfirmEmailUtil.java (hunk 1 above)
Comment (dbojanek) [reply dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/mail/confirmemail/ConfirmEmailUtil.java:57]: They both work correctly. The distinction is: email notifications are specific to language specified by user in his account, while notifications in the app itself are based on currently selected language.

Hence we need both methods: the one with locale is for emails, the one without is for app notifications.

File: dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/BrandingUtil.java (hunk 2 above)
Comment (dbojanek) [reply dataverse-persistence/src/main/java/edu/harvard/iq/dataverse/common/BrandingUtil.java:27]: They both work correctly. The distinction is: email notifications are specific to language specified by user in his account, while notifications in the app itself are based on currently selected language.

Hence we need both methods: the one with locale is for emails, the one without is for app notifications.

**Review by MrK191 — APPROVED** [review]

**Pull request merged**